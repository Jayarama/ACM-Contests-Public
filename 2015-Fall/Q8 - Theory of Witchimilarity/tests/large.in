500
Wilbur 14 59 31 pastel_green
Hodgdon 94 66 52 raw_umber
Bernita 87 98 54 ginger
Elem 51 76 64 khaki_html_css_khaki
Towery 68 90 59 orange_web_color
Jawad 39 16 58 persian_orange
Batty 37 67 58 burgundy
Willard 4 7 21 medium_persian_blue
Klara 9 3 73 cotton_candy
Kopacz 11 22 75 white_smoke
Jamison 16 35 36 royal_blue_web
Erich 65 58 86 kelly_green
Frank 63 94 58 falu_red
Rispoli 35 81 51 indigo
Emerald 52 70 61 sunglow
Quinton 84 66 15 fashion_fuchsia
Willer 73 29 68 brandeis_blue
Tabarez 14 60 51 lavender_pink
Ingle 87 55 98 deep_chestnut
Corpus 73 87 83 maize
Jepsen 85 33 83 lawn_green
Rosalba 26 15 37 plum_web
Tullis 23 52 42 classic_rose
Candida 4 26 62 pink
Elwanda 71 76 59 persimmon
Latashia 27 96 96 red_violet
Donetta 36 99 84 cadet_grey
Coralie 70 92 3 phlox
Messing 71 67 28 zinnwaldite_brown
Kapinos 42 80 59 carmine_m_p
Vonnie 22 68 60 lavender_magenta
Kidder 62 90 97 rifle_green
Orman 30 72 14 carnation_pink
Signe 93 44 33 han_blue
Sylvest 3 48 13 powder_blue_web
Jeffie 32 32 75 dark_raspberry
Deandra 86 95 70 pink_orange
Woolsey 21 90 19 sandstorm
Gregg 57 2 81 dandelion
Veselka 72 43 20 rich_lilac
Nannette 84 26 46 pink_orange
Clyburn 2 95 47 lemon
Cottrell 35 11 67 boston_university_red
Shirlene 29 38 66 yellow
Lippard 39 13 37 heliotrope
Moshe 70 7 78 medium_vermilion
Provenzano 90 23 99 blue_ryb
Jessie 73 41 17 lilac
Barmore 50 63 36 manatee
Pedro 83 25 40 hollywood_cerise
Ezekiel 26 83 28 sandy_taupe
Waggener 68 54 16 islamic_green
Thurber 70 26 56 deep_magenta
Mysliwiec 70 70 10 battleship_grey
Jeannine 67 34 96 chrome_yellow
Robin 17 13 8 rose_pink
Anitra 63 82 45 pale_plum
Overturf 2 52 45 ucla_blue
Patrice 19 23 17 chocolate_traditional
Despina 90 72 2 deep_magenta
Delia 22 13 28 dark_cerulean
Abram 83 60 7 bulgarian_rose
Kisha 29 94 79 caf_au_lait
Vincenza 44 100 100 thulian_pink
Angelo 26 92 88 old_lace
Ligon 65 17 12 caribbean_green
Partin 4 95 99 robin_egg_blue
Lucile 3 3 51 feldgrau
Wis 47 53 32 gainsboro
Johana 9 18 39 dark_tangerine
Samayoa 77 65 6 khaki_html_css_khaki
Dennise 15 18 31 bright_turquoise
Gunderson 23 36 29 bubble_gum
Len 85 48 74 columbia_blue
Burges 39 100 88 vivid_tangerine
Boerger 66 26 96 wine
Saathoff 95 21 68 deep_jungle_green
Thomasine 53 37 75 lavender_rose
Rayos 54 76 72 yellow_munsell
Dawe 80 56 42 opera_mauve
Richard 9 17 5 red_munsell
Carlyle 82 90 77 neon_carrot
Terrilyn 24 10 9 cambridge_blue
Jeanne 42 18 65 green_html_css_green
Lobo 41 87 91 cerulean_blue
Slyvia 68 53 64 sienna
Sixta 37 58 59 royal_blue_web
Wanetta 40 38 10 halay_be
Hugh 62 45 1 antique_brass
Daniele 52 14 23 turkish_rose
Tandy 37 15 98 sky_blue
Wilks 53 74 13 ash_grey
Patch 8 54 78 medium_vermilion
Kam 82 96 100 royal_blue_traditional
Tibbitts 100 16 100 ruddy_brown
Rufina 31 84 59 wisteria
Jerry 96 82 38 desert
Moriah 65 38 16 caput_mortuum
Burkitt 98 24 29 capri
Briana 89 13 1 medium_spring_bud
Johna 40 28 55 dark_raspberry
Hollis 71 84 80 psychedelic_purple
Clara 32 19 43 rackley
Clyde 61 61 85 verdigris
Terra 35 5 50 dandelion
Garth 99 8 41 light_apricot
Ellenwood 26 91 72 moonstone_blue
Bryer 63 16 28 ivory
Deas 30 2 26 sunglow
Stults 87 77 70 medium_taupe
Janessa 64 37 23 dark_pastel_green
Transue 49 56 40 portland_orange
Nikole 41 51 22 sea_green
Kaczynski 49 52 41 twilight_lavender
Vannesa 9 94 31 selective_yellow
Jamee 52 55 7 chamoisee
Rattler 92 74 28 old_lavender
Penix 21 5 23 ash_grey
Kohnke 23 51 40 boysenberry
Charla 18 56 23 burnt_sienna
Tisby 44 41 63 bittersweet
Earlean 56 26 68 naples_yellow
Govea 31 88 45 buff
Ellerman 16 96 65 ruddy_pink
Maurice 14 8 83 dark_taupe
Almon 79 92 50 ruby_red
Jade 11 66 9 pistachio
Poulsen 99 20 98 melon
Sulema 58 2 58 vanilla
Elicia 41 51 89 fallow
Luse 66 50 73 pakistan_green
Peltz 98 38 85 manatee
Marylyn 97 68 31 indigo_web
Genevive 8 51 25 verdigris
Shawgo 47 72 63 cadmium_green
Catlett 69 39 5 rose_vale
Margarita 84 59 26 slate_blue
Viers 53 37 47 pale_plum
Grooms 43 26 39 pale_cerulean
Lupien 50 73 37 french_raspberry
Tressa 1 79 44 battleship_grey
Fae 53 94 93 royal_purple
Mulligan 41 38 91 university_of_california_gold
Reimers 37 76 20 lemon_lime
Mclellan 41 34 99 cosmic_latte
Marquette 79 34 4 fuchsia_pink
Messana 31 69 71 ultramarine_blue
Asley 92 62 81 mountain_meadow
Pura 61 67 78 dark_pastel_purple
Charley 86 90 38 deep_carmine_pink
Loretta 92 43 90 cobalt
Bonny 57 50 8 cadmium_green
Carlton 63 18 21 bright_green
Schwalm 43 28 19 bazaar
Raymond 79 52 73 mantis
Whelpley 30 44 42 dark_pastel_green
Schmalz 50 66 10 rose_gold
Hobert 68 1 20 bittersweet_shimmer
Broom 97 2 16 ash_grey
Jeppson 8 35 32 sacramento_state_green
Chiodo 75 71 56 aurometalsaurus
Pascua 89 19 10 ultramarine
Lorine 15 26 62 medium_violet_red
Laurent 44 74 18 medium_spring_green
Disanto 91 24 29 caribbean_green
Jiles 20 32 81 prussian_blue
Beula 57 16 37 coffee
Keele 79 20 88 antique_fuchsia
Regena 32 98 15 turquoise
Stefan 31 31 99 copper_crayola
Graff 52 33 47 rose_bonbon
Curt 92 19 75 vivid_cerise
Paiz 53 71 89 medium_sea_green
Slabaugh 72 99 84 tan
Leta 68 71 11 cinereous
Gianna 20 7 75 cherry
Hurwitz 86 60 49 verdigris
Mantilla 63 35 67 violet_web
Hedrick 75 29 64 timberwolf
Ashlock 25 97 40 pink_pearl
Timm 94 21 33 dark_goldenrod
Parisi 36 19 37 dark_jungle_green
Caruana 53 23 5 ultramarine_blue
Valeria 67 93 83 electric_yellow
Kym 72 25 35 halay_be
Guercio 53 77 64 emerald
Brook 93 23 41 school_bus_yellow
Cherry 13 44 85 salmon
Dorsey 18 23 57 true_blue
Britni 59 46 57 boysenberry
Roy 68 50 44 brandeis_blue
Streit 7 74 66 cg_blue
Obyrne 70 34 14 venetian_red
Patrina 77 68 20 cinnamon
Diego 87 51 86 powder_blue_web
Tarbell 29 69 89 deep_peach
Piazza 22 35 84 persian_indigo
Branda 50 42 76 daffodil
Idell 60 63 83 ruddy_pink
Massenburg 32 58 32 thulian_pink
Doria 29 3 85 dark_electric_blue
Melba 58 71 3 carnelian
Arnold 2 79 1 olive_drab_web_olive_drab_3
Lane 71 98 72 persian_orange
Burnes 50 24 88 gold_metallic
Amado 43 75 84 light_crimson
Marya 82 35 86 rich_maroon
Willene 21 5 41 indigo_dye
Ciara 100 1 13 non_photo_blue
Sennett 2 7 94 light_sky_blue
Louie 37 41 18 halay_be
Mullings 76 22 41 deep_lilac
Katharina 35 20 61 nadeshiko_pink
Valenza 89 78 20 tea_rose_orange
Lashley 77 34 98 sea_green
Bob 16 13 83 dim_gray
Schenkel 4 40 32 spiro_disco_ball
Luppino 39 88 46 fuchsia_rose
Pilgrim 23 94 24 deep_lilac
Valene 94 26 17 laurel_green
Archie 85 26 66 medium_blue
Rothfuss 5 83 77 dark_lava
Lavern 70 15 21 antique_brass
Clifford 15 99 82 light_fuchsia_pink
Gleason 59 19 41 pale_lavender
Stidham 13 98 55 imperial_blue
Ridinger 90 28 22 shadow
Lemley 12 8 8 meat_brown
Ahmed 89 94 93 lemon
Bettinger 88 54 8 lime_web_x11_green
Lida 47 61 97 opera_mauve
Leonia 92 69 75 light_gray
Lurlene 24 94 12 debian_red
Mehr 28 63 8 ruddy
Tavernier 9 75 25 tufts_blue
Morrill 10 12 3 bright_cerulean
Flakes 72 60 73 candy_apple_red
Korey 82 4 98 neon_carrot
Hagans 91 5 42 wheat
Smyre 41 68 69 jade
Nola 34 68 73 la_salle_green
Eskew 18 100 6 moss_green
Melani 49 10 79 redwood
Siu 6 9 41 maize
Lor 24 55 59 persian_red
Marriner 86 98 76 mode_beige
Nelson 70 35 90 sea_blue
Marco 40 89 1 tan
Friedlander 39 100 21 palatinate_purple
Paris 5 15 14 british_racing_green
Dyan 61 90 84 white_smoke
Dorine 25 91 44 tea_rose_rose
Wagaman 96 96 88 skobeloff
Weathers 16 72 92 guppie_green
Jayne 35 56 63 canary_yellow
Rocco 38 48 85 linen
Ogren 77 10 86 dark_terra_cotta
Worth 100 74 2 deep_tuscan_red
Garoutte 11 99 81 blue_ryb
Jenee 22 22 17 burgundy
Yanni 84 23 28 quartz
Moak 46 57 83 light_apricot
Dufault 33 54 6 french_raspberry
Fuller 58 80 35 golden_yellow
Whitehead 47 13 9 thulian_pink
Blaschke 34 88 91 medium_persian_blue
Zina 65 11 16 spring_green
Pelletier 65 21 32 magenta_dye
Maire 85 31 18 bleu_de_france
Magdalen 92 84 92 sap_green
Andujar 63 60 20 asparagus
Wallen 97 46 15 arylide_yellow
Anneliese 9 71 41 battleship_grey
Harshberger 38 39 93 peridot
Astin 57 79 54 sap_green
Dotson 16 57 64 pale_carmine
Jeanna 67 39 12 copper_crayola
Burnam 81 74 21 sunglow
Nick 10 53 95 pale_magenta
Shaffer 5 85 46 french_lime
Huisman 32 42 60 taupe
Brundidge 92 43 60 mauvelous
Ochs 32 41 98 bisque
Bernett 8 95 10 meat_brown
Brianne 18 5 94 green_color_wheel_x11_green
Kiesha 82 96 74 wenge
Cohan 41 52 69 neon_fuchsia
Schwager 97 2 79 old_mauve
Brathwaite 91 81 35 auburn
Enola 81 73 22 opera_mauve
Bernardino 25 81 18 olive_drab_web_olive_drab_3
Marietta 88 28 97 burnt_orange
Kina 87 65 10 flax
Ralph 4 92 32 violet_web
Sandy 70 35 73 olive_drab_web_olive_drab_3
Labelle 32 43 84 buff
Burpee 57 29 48 dark_lava
Bonita 92 83 64 mode_beige
Tousant 36 98 16 eggshell
Blevins 51 21 8 opera_mauve
Andreas 44 22 71 tenn_tawny
Elna 29 99 13 pale_copper
Willow 10 47 22 deep_saffron
Mack 39 94 6 battleship_grey
Aleida 10 79 80 caf_noir
Carroll 1 82 14 crimson_glory
Marcene 15 25 47 beau_blue
Constance 53 56 39 denim
Sackrider 17 79 3 gray_x11_gray
Veta 64 6 6 pale_gold
Rayner 5 31 20 blue_green
Raynes 48 77 46 golden_poppy
Carrales 59 47 64 lincoln_green
Bluhm 60 65 23 light_yellow
Heike 83 69 24 coquelicot
Grisby 41 48 100 dogwood_rose
Sudie 3 60 9 eggshell
Clint 38 81 57 dark_terra_cotta
Tama 52 89 1 green_yellow
Maragret 12 56 45 byzantium
Latrisha 64 20 65 candy_apple_red
Jana 29 93 31 tenn_tawny
Seitz 56 12 38 electric_crimson
Villar 32 66 46 celadon_blue
Birch 6 35 87 copper_rose
Slevin 54 25 64 bole
Platero 72 26 53 peach
Saragosa 12 23 2 tangelo
Kacy 82 41 18 la_salle_green
Keri 65 94 67 light_taupe
Shirl 26 31 53 pale_spring_bud
Barnhill 61 18 61 charcoal
Howey 87 31 19 bisque
Sandie 35 24 29 purple_html_css
Shavonda 21 56 50 pastel_blue
Buhr 46 80 72 azure_mist_web
Bethel 9 85 84 lime_web_x11_green
Tuthill 11 40 16 blue_gray
Merrill 13 70 76 light_goldenrod_yellow
Deana 7 56 75 north_texas_green
Ashely 39 100 32 dark_yellow
Marcos 1 32 29 blue_violet
Alexander 53 49 72 cerise
Lebron 67 80 68 golden_poppy
Gunning 74 62 86 dark_byzantium
Corrina 5 66 38 burgundy
Gonyea 95 30 7 alabama_crimson
Mari 37 73 10 medium_carmine
Livsey 46 13 50 baby_blue
Brumbelow 6 65 1 midnight_blue
Genevie 37 41 81 ufo_green
Storch 66 52 40 baby_blue
Lavina 53 30 98 cerise_pink
Orloff 69 70 39 harvard_crimson
Edwin 35 46 61 pale_green
Holley 7 44 71 lemon_chiffon
Mcnaughton 34 54 62 davy_s_grey
Iluminada 95 50 52 black_olive
Zumwalt 54 27 76 papaya_whip
Roxanne 92 61 75 chocolate_web
Vanesa 68 72 87 bronze
Pierre 56 70 55 rifle_green
Ngo 57 7 58 electric_lavender
Osullivan 67 83 44 pastel_blue
Kirshner 71 87 75 pink
Clora 95 74 4 celadon
Stepanie 47 5 70 golden_poppy
Particia 48 62 62 light_salmon
Mcclain 94 35 15 midnight_blue
Herb 53 25 98 mauvelous
Marine 54 66 9 carmine_red
Fermina 69 11 20 big_dip_o_ruby
Delinda 82 48 10 electric_ultramarine
Frier 68 21 90 gray_html_css_gray
Walthall 39 29 65 cadet
Cutlip 45 44 47 eggplant
Dillon 33 82 99 outrageous_orange
Mclain 92 98 98 bright_turquoise
Bessie 37 84 90 iris
Dirk 47 79 23 steel_blue
Czech 1 84 38 ucla_gold
Dagmar 3 49 79 selective_yellow
Lightle 60 99 83 arylide_yellow
Roderick 8 54 63 lemon
Irwin 95 56 3 celestial_blue
Irmgard 64 33 55 brilliant_lavender
Heras 17 81 13 red_brown
Kimberely 52 52 45 mardi_gras
Loper 73 70 3 thistle
Arellano 82 28 32 french_blue
Santoyo 9 43 76 green_pigment
Hyun 1 71 34 green_crayola
Seely 2 38 96 rosso_corsa
Katharyn 45 81 5 straw
Ying 20 18 3 carmine_pink
Florencio 85 65 6 dark_sea_green
Mason 35 8 7 forest_green_web
Moorer 49 92 8 hunter_green
Wegner 54 6 56 limerick
Syreeta 76 12 42 medium_champagne
Eleanore 22 97 14 chartreuse_traditional
Leigh 12 18 55 peach_orange
Mercado 18 78 95 cordovan
Wittner 51 82 61 aquamarine
Leishman 53 15 27 persian_orange
Noelle 38 91 2 forest_green_web
Valrie 24 98 55 indigo_web
Janice 96 23 48 light_blue
Kail 72 21 61 laser_lemon
Adrienne 49 91 100 bone
Janise 2 42 90 yellow_munsell
Royall 65 48 84 pink
Myrl 34 99 61 deep_tuscan_red
Dublin 61 24 7 electric_blue
Steinhauser 53 73 57 wenge
Ladue 78 32 63 copper_rose
Brittanie 96 20 6 aqua
Jeanene 26 71 96 sky_magenta
Sonja 87 40 59 pastel_red
Chaffins 72 24 11 cyan
Elizabeth 3 91 91 light_cornflower_blue
Violet 68 66 45 pale_taupe
Deirdre 7 80 54 lavender_rose
Sigrid 44 93 72 blue_crayola
Maragaret 70 52 99 pakistan_green
Sharmaine 74 61 37 gold_metallic
Delcambre 9 2 28 tiffany_blue
Seipel 80 52 30 baby_pink
Demas 67 88 83 burnt_orange
Dina 60 27 15 sea_blue
Marianne 81 99 9 floral_white
Fenderson 8 59 18 amethyst
Trumbauer 70 72 53 medium_tuscan_red
Hedy 15 85 98 topaz
Christie 79 6 53 blizzard_blue
Marcela 88 29 53 wheat
Shannon 2 18 99 vanilla
Romberg 80 31 99 copper_penny
Brouillard 85 79 7 rosy_brown
Selander 79 70 45 violet_ryb
Deann 90 25 25 malachite
Lean 90 55 88 blue_violet
Nina 21 91 45 blue_ryb
Suk 74 21 86 blue_pigment
Torrie 13 42 8 electric_green
Stalnaker 17 24 7 rich_lavender
Clickner 1 48 29 kenyan_copper
Dominique 60 71 38 medium_jungle_green
Kirven 40 53 34 mikado_yellow
Tyson 61 39 26 copper_rose
Wicks 40 31 96 medium_violet_red
Genna 57 19 71 dark_tangerine
Jozwiak 86 22 93 canary_yellow
Fleshman 79 10 22 tango_pink
Dollie 31 84 35 lime_web_x11_green
Shanelle 73 55 75 asparagus
Carter 70 7 14 alabama_crimson
Nada 19 18 5 deep_fuchsia
Soller 54 65 72 copper_red
Lashunda 67 77 27 copper_penny
Marion 71 21 81 dark_tangerine
Dann 90 7 78 ua_red
Colpitts 73 7 4 honolulu_blue
Connors 10 48 12 wenge
Mitsue 83 20 33 shamrock_green
Carly 66 41 16 pale_green
Delahoussaye 2 12 6 tenn_tawny
Parchman 41 94 87 celestial_blue
Su 71 74 89 international_klein_blue
Czajkowski 10 4 88 fuchsia_rose
Alisia 98 15 17 lime_green
Rutter 20 99 93 dark_pink
Delicia 86 40 10 pistachio
Phu 93 42 90 field_drab
Svoboda 42 16 68 lime_color_wheel
Shelly 55 85 33 urobilin
Rego 17 50 74 bittersweet
Lafrance 89 97 98 indigo
Zynda 75 87 8 dark_electric_blue
Coy 2 4 59 sandy_brown
Beata 77 15 57 peach
Stecker 46 100 29 moonstone_blue
Binette 2 43 84 cherry
Waguespack 51 69 11 bleu_de_france
Mariela 36 88 37 purple_mountain_majesty
Thea 100 79 89 ku_crimson
Vanarsdale 47 97 90 light_pink
Kotter 48 4 37 scarlet
Jamaal 100 38 43 outrageous_orange
Mccotter 2 93 15 english_lavender
Charlie 37 49 13 fuchsia_rose
Fenster 95 98 14 antique_ruby
Edna 31 52 18 french_beige
Lamanna 5 97 38 coral_red
Sachiko 19 17 24 coffee
Lance 86 74 82 desert
Marjory 71 31 23 cotton_candy
Whicker 49 5 98 orange_red
Scioneaux 16 75 85 jet
Niki 94 8 68 flamingo_pink
