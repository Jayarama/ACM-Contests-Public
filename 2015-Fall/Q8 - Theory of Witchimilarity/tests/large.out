Alexander
Binette
Birch
Bob
Branda
Brianne
Candida
Cherry
Czajkowski
Dagmar
Deana
Edwin
Elicia
Genevie
Grisby
Harshberger
Herb
Holley
Huisman
Janise
Jeanene
Jeannine
Jeffie
Jiles
Kopacz
Labelle
Lavina
Lida
Lorine
Maragaret
Maurice
Mclellan
Mulligan
Nick
Ochs
Patch
Piazza
Rego
Rocco
Roderick
Royall
Santoyo
Seely
Sennett
Shannon
Shirlene
Stefan
Tandy
Weathers
Wicks
